src/Bookings.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
```

Comment:
```suggestion
import fakeBookings from "./data/fakeBookings.json";
```
Keep this variable starting with a lowercase - I would only capitalise variables if they represent a React component